pages/portfolio3.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="../styles/style-portfolio.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <main class="grid-parent">
+    <header>
+      <h1 class="title">
+        Camilla Berretta, full-stack developer
+      </h1>
+    </header>
+    <section>
+    </section>
+
+    <article>
+      <h1>My Portfolio</h1>
```

Comment:
You can use a different header tag here since you've already got a h1 element on the page.